.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=false
```

Comment:
Libraries (as opposed to applications like the `balena-builder` or `balena-cli`) should use lock files.